Source/Extensions/Blazorise.Components/ListView.razor

```diff
@@ -8,4 +8,8 @@
             <ListGroupItem Name="@text">@text</ListGroupItem>
         }
     }
+    @if ( ChildContent is not null )
```

Comment:
I don't think we need a null check. It will be either rendered or not. You can just set `@ChildContent`

Reply:
You know what? I get the feeling I do this everytime haha Will change.

Reply:
I'm not sure I think I've found some unconventional ways it'll throw. Like when you manually build or compose components.

Also, what happens when you have both 
`ChildContent`
`AnotherContent`

And you only do
```
<ListView>
<AnotherContent>
</AnotherContent>
<ListView/>
```

Reply:
Nothing will happen in that case. The problem is when you try to mix the content without specifying ChildContent. eg.

```
<ListView>
  <AnotherContent>
  </AnotherContent>
  <div>some other content</div>
<ListView/>
```

In this case, Blazor will complain. The solution is to do:

```
<ListView>
  <AnotherContent>
  </AnotherContent>
  <ChildContent>
    some other content
  </ChildContent>
<ListView/>
```